mithril.js

```diff
@@ -1706,16 +1706,16 @@
 		var currentTarget = e.currentTarget || e.srcElement
 		var args
 
+		while (currentTarget && !/^a$/i.test(currentTarget.nodeName)) {
```

Comment:
@barneycarroll Probably not as much of an edge case as you might immediately think. Also, keep in mind this is for `config: m.route` (`<a>` is actually the most common case for this). Here's a list of elements that can have an `href` attribute:

- `<a>` - you hopefully should know this by now.
- `<area>` - denotes the target address for a clickable area within an image map, much like `<a>`.
- `<base>` - only in `<head>`, denotes the base URL for which all relative links are resolved to.
- `<link>` - you should also hopefully know this one, but do note this is technically a metadata element, and thus does not affect behavior on its own (CSS can, though).

Out of all of these, only `<a>` and `<area>` really mean anything to the router, since those are clickable hyperlinks. It should only allow those two, case insensitive. A technically correct regexp would be `/^a(rea)?$/`, and the original was already mostly acceptable.